src/app/settings/shelf-settings/components/static-buttons-configuration-card/static-buttons-configuration-card.component.html

```diff
@@ -1,14 +1,12 @@
 <sofie-card class="pt-2 pl-0">
     <ng-container cardHeader>
-        <span class="card-header" i18n>static-buttons-configuration.card-header</span>
+        <span i18n>static-buttons-configuration.card-header</span>
         <sofie-icon-button [icon]="Icon.PEN" [iconSize]="IconSize.M" [tooltip]="'Edit Static Buttons'" (click)="openEditStaticButtons()"></sofie-icon-button>
     </ng-container>
         <div class="static-actions-display">
-            <ng-container *ngIf="selectedStaticActions.length !== 0 else noStaticButtons">
-                <div class="action-container">
-                    <sofie-action-box *ngFor="let action of selectedStaticActions" [action]="action"></sofie-action-box>
-                </div>
-            </ng-container>
+            <div class="action-container" *ngIf="selectedStaticActions.length !== 0 else noStaticButtons">
+                <sofie-action-box *ngFor="let action of selectedStaticActions" [action]="action"></sofie-action-box>
```

Comment:
```suggestion
                <sofie-action-box *ngFor="let action of selectedStaticActions" [action]="action" />
```

Just a suggestion :) Let me quickly find the rule to prefer self-closing, if you want to implement that

Reply:
I made a PR where the rule is in now :)